### ISSUE TITLE
BootstrapBlazor.DataAcces.EntityFrameworkCore DefaultDataService not search corectly

### ISSUE THREAD

**Issue by franklupo** [dotnetcore/BootstrapBlazor#182]
### Describe the bug
When searching for a number it is not searched in the numeric property

### To Reproduce
```c#
@inject IDataService<Blog> DataServiceBlog;

<PageTitle>Index</PageTitle>

<Table TItem="Blog" DataService="@DataServiceBlog"
       AllowResizing="true" IsStriped="true" IsBordered="true"
       ShowToolbar="true" ShowDefaultButtons="true" ShowSearch="true"
       ShowAdvancedSearch="false" IsMultipleSelect="true"
       ShowSkeleton="true" IsPagination="true"  ShowSearchTextTooltip="false"
       ShowExtendButtons="false">
    <TableColumns>
        <TableColumn @bind-Field="@(context.Url)"  Searchable="true"/>
        <TableColumn @bind-Field="@(context.Rating)" Searchable="true" />
        <TableColumn @bind-Field="@(context.Required)" Searchable="true"/>
    </TableColumns>
</Table>

ublic class BloggingContext : DbContext
{
    public BloggingContext(DbContextOptions options) : base(options) { }
    protected BloggingContext() { }

    public DbSet<Blog> Blogs { get; set; }
    public DbSet<Post> Posts { get; set; }

    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
    {
        optionsBuilder.UseSqlite($"Data Source=db.db");
    }
}

public class Blog
{
    [Key]
    public int Id { get; set; }

    [Required]
    public string Url { get; set; }
    public int Rating { get; set; }
    public bool Required { get; set; }
    public List<Post> Posts { get; set; }
}

public class Post
{
    [Key]
    public int Id { get; set; }
    public string Title { get; set; }
    public string Content { get; set; }

    public Blog Blog { get; set; }
}

```

### Further technical details
- BootstrapBlazor" Version="6.5.19"
- BootstrapBlazor.DataAcces.EntityFrameworkCore" Version="6.0.0"

**Comment by ArgoZhang** [comment]
@franklupo hi. can you create a repo for this issue? i need the repro environment

**Comment by franklupo** [comment]
https://paste.c-net.org/SnackJoanna

**Comment by densen2014** [comment]
At present, there is a problem with ef's data service matching number type search,
or you can try it with fsql

[BlazorApp1WithFsql.zip](https://github.com/dotnetcore/BootstrapBlazor/files/8610809/BlazorApp1WithFsql.zip)

**Comment by ArgoZhang** [comment]
@franklupo hi. this is bug. I'm work on it.

**Comment by franklupo** [comment]
very good

**Comment by ArgoZhang** [comment]
@franklupo 

for now the Table component only support string data type column for search. so all types need to be supported. could you submit a PR for this Issue? Or wait for me to fix this issue.

https://github.com/dotnetcore/BootstrapBlazor/blob/0db64308629e7a597195b111343288f049697e89/src/BootstrapBlazor/Components/Table/Table.razor.Search.cs#L287

**Comment by ArgoZhang** [comment]
@franklupo hi fixed and bump version 6.6.3. you can have a try for this.
